lib/ocpp/v201/charge_point.cpp

```diff
@@ -1198,6 +1198,23 @@ void ChargePoint::handle_scheduled_change_availability_requests(const int32_t ev
     }
 }
 
+/**
+ * Determine for a component variable whether it affects the Websocket Connnection Options (cf.
```

Comment:
```suggestion
 * Determine for a component variable whether it affects the Websocket Connection Options (cf.
```

Can you install a spellchecker, please? E.g. https://github.com/streetsidesoftware/vscode-spell-checker

I mean I wouldn't have seen it, too, if I wouldn't have this extension installed :slightly_smiling_face:

Reply:
ah, yes indeed... my problem is rather that my IDE already spell checks, it was just too easy to ignore it - I added a pre-commit hook now ;)